plugins/home/src/homePageComponents/QuickStart/ContentModal.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { Link } from '@backstage/core-components';
+import Modal from '@material-ui/core/Modal';
+import Box from '@material-ui/core/Box';
+
+import { useStyles } from './styles';
+
+export type ContentModalProps = {
+  modalContent: React.JSX.Element;
+  linkContent: string | React.JSX.Element;
+};
+
+export const ContentModal = (props: ContentModalProps) => {
+  const { modalContent, linkContent } = props;
+  const styles = useStyles();
+  const [open, setOpen] = useState(false);
+
+  return (
+    <div className={styles.linkText} data-testid="content-modal-container">
+      <Link
+        to="#"
```

Comment:
is it possible to skip this property when the component "button" is used?

Reply:
I'm not sure I understand the question 🤔 
It looks like the prop `to` is required in the [LinkProps definition](https://github.com/backstage/backstage/blob/7d8777dbba10cabe0629e8f239806231519f5062/packages/core-components/src/components/Link/Link.tsx#L103).
Should I use something other than `Link`?

Reply:
Oh my bad, didn't realised this was the Link component from backstage core components instead of mui. I'll continue to review your changes during the day!